src/omlt/io/keras_reader.py

```diff
@@ -0,0 +1,42 @@
+from omlt.neuralnet.network_definition import NetworkDefinition
+from omlt.neuralnet.layer import InputLayer, DenseLayer
+
+def load_keras_sequential(nn, scaling_object=None, scaled_input_bounds=None):
+    """
+    Load a keras neural network model (built with Sequential) into
+    a pyoml network definition object. This network definition object
+    can be used in different formulations.
+    Parameters
+    ----------
+    nn : keras.model
+        A keras model that was built with Sequential
+    scaling_object : instance of object supporting ScalingInterface (see scaling.py)
+    scaled_input_bounds: list of tuples
```

Comment:
`scaled_input_bounds` is a dictionary now. we can probably copy from `network_definition.py`

Reply:
fixed